compiler/damlc/tests/daml-test-files/TryImplementNonInterface.daml

```diff
@@ -0,0 +1,18 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- @SINCE-LF-FEATURE DAML_INTERFACE
+-- @ERROR range=18:7-18:20; Tried to get an interface view of type ‘MyUnit’ from type ‘Bool’
```

Comment:
Sorry I wasn't clear, I was referring to this error
```haskell
displayError info TriedImplementView { target, triedReturnType, expectedReturnType } =
  pure $ text "Tried to implement a view of type" <+> pprq triedReturnType <+> text "on interface" <+> pprq target
      <> text ", but the definition of interface" <+> pprq target <+> text "requires a view of type" <+> pprq expectedReturnType
```
https://github.com/digital-asset/ghc/blob/53eee06bb058ea88d8ec11638e5ef1de653ff2fa/compiler/typecheck/TcDaml.hs#L139-L141

Reply:
sorry, I should've realized :sweat_smile: done